README.md

```diff
@@ -416,6 +416,38 @@ server.addHandler( new FileFallbackHandler(_fs, "/jquery/jq1.11.1.js" , "/jquery
 server.addHandler( new FileFallbackHandler(_fs, "/jquery/jqm1.4.5.js" , "/jquery/jqm1.4.5.js" ,   "http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js",  "max-age=86400"));
 ```
 
+### Serving static files from SPIFFS
+Serve a specific file or attach a folder in SPIFFS to a root url.
+An optional Cache-Control header can be added to responses.
+An optional Last-Modified header can be added, if the request contains If-Modified-Since header with the same value the server will respond with 304 code instead of serving the file.
+```cpp
+
+// Serve static file example
+server.serveStatic("/url/file.htm", SPIFFS, "/dir/file.htm");
+// Serve static folder example
+server.serveStatic("/root-url", SPIFFS, "/root-dir");
+// Specify cache control
+server.serveStatic("/root-url", SPIFFS, "/root-dir", "max-age=86400");
+// Specify last modified (should be updated each time files are updated)
+server.serveStatic("/root-url", SPIFFS, "/root-dir", NULL, "Wed, 08 Jun 2016 22:00:00 GMT");
+
+```
+
+### HTTP Redirect (302)
+Return HTTP 302 response to redirect the client to a different location.
+An optional IPAddress can specify redirection exclusions, for example, exclude redirection to requests to the AP address.
+```cpp
+// Redirect all requests
+server.redirect("/root/file.ext", "/other-location/other-file.ext");
+// Redirect requests not for the AP address
```

Comment:
the redirect by IP for files seems to propose the same functionality as the FileFallbackHandler documented above.
You can see that it also has configuration for which file system to use, because the server can run from SD or other storage.
Another note on redirecting to local files like this is that you require another request to be made and parsed by the server, while you can serve the local file with the first one.
I agree that redirecting to URLs is a good idea, but let's not document that as redirecting to files like this (for CDN) and use the FileFallbackHandler for handling that and document other use cases where redirection makes more sense and does not cause extra requests to be sent to the ESP if they can be avoided :)
FileFallbackHandler could be maybe better exposed through server.serveCDN().
I feel that I need to brainstorm a bit more on the redirect thing. I'm all up for the serveStatic changes for sure, though maybe you should think of a way to update the modification date after the sketch is started (and the file is updated)